c10/macros/Macros.h

```diff
@@ -212,7 +212,7 @@ constexpr uint32_t CUDA_THREADS_PER_BLOCK_FALLBACK = 256;
 // CUDA_KERNEL_ASSERT checks the assertion
 // even when NDEBUG is defined. This is useful for important assertions in CUDA
 // code that when building Release.
```

Comment:
While you are here, could you please also fix the comment above your change? For example

> CUDA_KERNEL_ASSERT checks the assertion even when NDEBUG is defined. This is useful for important assertions in CUDA code that **would otherwise be suppressed** when building Release.